docs/scripts/buildApi.ts

```diff
@@ -156,7 +156,11 @@ async function annotateComponentDefinition(context: {
 
   const demos = uniqBy<ReactApi['pagesMarkdown'][0]>(
     api.pagesMarkdown.filter((page) => {
-      return page.components.includes(api.name);
+      return (
+        page.components.includes(api.name) ||
```

Comment:
These changes are required for generating the `Demo:` jsdocs block for the unstyled components.